### PR TITLE
New Login Design

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-03-04 at 3 13 44 PM
Screen Shot 2022-03-07 at 1 02 37 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by oscarocastellanos — DISMISSED** [review]
LGTM

File: swagger-operator.yml [hunk 1]
```diff
@@ -2838,26 +2855,11 @@ definitions:
...
+  checkOperatorVersionResponse:
     type: object
     properties:
-      key:
-        type: string
-      value:
+      current_version:
         type: string
+      latest_version:
+        type: string
```
Comment (jinapurapu) [inline swagger-operator.yml:2865]:
```suggestion
        type: string
        
```

File: swagger-console.yml [hunk 2]
```diff
@@ -4446,3 +4463,10 @@ definitions:
         type: string
       shortName:
         type: string
+  checkVersionResponse:
+    type: object
+    properties:
+      current_version:
+        type: string
+      latest_version:
+        type: string
```
Comment (jinapurapu) [inline swagger-console.yml:4472]:
```suggestion
        type: string
        
```

**Review by jinapurapu — CHANGES_REQUESTED** [review]

**Review by jinapurapu — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
Do we need to modify all these files (879)?

**Review by adfost — DISMISSED** [review]

**Review by Alevsk — APPROVED** [review]
tested, works fine

**Review by adfost — APPROVED** [review]
Tested, LGTM. However, I was just wondering, should this be further split into 2 PRs? You seem to have changed a lot of utils from cluster.* to utils.*, in addition to changing the login.

**Review by bexsoft — APPROVED** [review]
Need to review the "Welcome to" tag as I think that was meant to be removed from the login pages. Beside of that LGTM

**Pull request merged**